sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitCoverageResultImporter.java

```diff
@@ -61,20 +59,14 @@
 
   private static final String WRONG_LINE_EXCEPTION_MESSAGE = "Line with number %s doesn't belong to file %s";
 
-  /**
-   * Instantiates a new php unit coverage result parser.
-   *
-   * @param context the context
-   */
-  public PhpUnitCoverageResultImporter(FileSystem fileSystem) {
-    this.fileSystem = fileSystem;
+  public PhpUnitCoverageResultImporter() {
+    super(PhpPlugin.PHPUNIT_COVERAGE_REPORT_PATH_KEY, "unit test coverage");
```

Comment:
@inverno I don't like that we have inconsistent way of sharing data between classes in inheritance chain: through fields and constructor